.github/CONTRIBUTING.md

```diff
@@ -31,7 +31,15 @@ Please post code and output as text ([using proper markup](https://guides.github
 5. Ensure the code complies with our coding guidelines (see below).
 6. Send that pull request!
 
-Please make sure you have [set up your username and email address](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup) for use with Git. Strings such as `silly nick name <root@localhost>` look really stupid in the commit history of a project.
+## Specific set up for documentation application
+
+1. You should have installed `node`(_we recommend using [nvm](https://github.com/nvm-sh/nvm)_) and `yarn`.
+   * Using `nvm` you can execute `nvm use`(reads _.nvmrc_ file) in the project root directory and follow the instructions to use the correct `node` version
+2. Install dependencies with `yarn install`.
+3. Run local development server with `yarn doc:dev`.
+4. Implement your changes.
+
+Please make sure you have [set up your username and email address](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup) for use with Git. Strings such as `silly nick name <root@localhost>` looks bad in the commit history of a project.
```

Comment:
This paragraph doesn't belong to the documentation section, it should go above it, right?

Reply:
Yeah, i'll put it up again